src/shared/components/Resources/defaultResourcePayloads.ts

```diff
@@ -7,6 +7,7 @@ export const DEFAULT_RESOURCE = {
 export const DISK_STORAGE = {
   '@context': {}, // optional
   '@id': '', // optional
+  '@type': ['DiskStorage'],
```

Comment:
Documentation says `@type: "DiskStorage"`. I know this works too but we might as well be consistent with the doc